actions/geoip.py

```diff
@@ -95,6 +100,13 @@ def run(self, ip_addresses):
                                       'value': response.isp}
                     details['org'] = {'name': "Org",
                                       'value': response.organization}
+                elif reader_asn:
```

Comment:
I did an `elf` as there seems to be an overlap of the information in the two databases, but the ISP has 2 additional fields. So if they have the ISP database it does not make sense to overwrite the possibility better information from ISP.